bin/ncar_build.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Assume that this script is run from the root directory
+
+# If an argument is supplied and if it's 'GPU' (case insensitive), assign that to the variable 
+#    in CMakeLists.txt file under the root directory. Default is CPU mode.
+MODE=${1^^}
+if [ "$MODE" == "GPU" ]; then
+  sed -i 's/MODE CPU/MODE GPU/g' CMakeLists.txt
+  # Somehow the Release mode will break the GPU code; change back to Debug mode
+  sed -i 's/CMAKE_BUILD_TYPE Release/CMAKE_BUILD_TYPE Debug/g' CMakeLists.txt
+  if [ "$NCAR_HOST" == "casper" ]; then
+    # Use cc70 for V100 GPU on Casper
```

Comment:
@sjsprecious Casper has both V100 and A100 nodes.  I believe that H100 nodes will be installed on Casper sometime in the summer.    We can't assume that we will only be using the V100 nodes on Casper

Reply:
Good point. I have removed this hardcode change so a user needs to revise the CMakeLists file manually for the desired GPU architecture.